addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1765,6 +1765,12 @@
 			"prefab"		"140"
 		}
 		
+		"589"	//The Eureka Effect
+		{
+			"desp"			"The Eureka Effect: {positive}+20 metal every 5 seconds, reduced construction speed on hit penalty"
```

Comment:
reducing construction on hit feels a bit redundant as it would do very little difference. But if were to keep it, should add `to 35%` at the end to mention exact valve change

Reply:
removed the construction on hit change